ui.apps/src/main/content/jcr_root/apps/guides-components/components/guidessidenavigation/clientlibs/site/js/guidessidenavigation.ts

```diff
@@ -109,14 +116,16 @@ class GuidesNavigation {
         }
     }
 
-    renderLevel(children: Array<any>, idx: number, parentPath: string, incrementer: number) {
-        if(!children) {
-            children = [];
+    doRenderLevel(ul, offset, currIdx, children, parentPath, idx, incrementer, button) {
```

Comment:
If we are not changing the incrementer then why are we passing this. shouldn't this be fixed and referred. is this used for css property ?

Reply:
it is incremented in certain places. yes, it is used to specify the depth of the item as a css class property.